docs/build-on-linea/linea-sdk/index.md

```diff
@@ -2,8 +2,6 @@
 title: Linea SDK
 ---
 
-The Linea SDK package is comprised of two integral components: the SDK, which facilitates smart contract interaction, and the Postman, responsible for seamless message delivery. Together, these components enable smooth communication and empower you to effortlessly retrieve message status and information across the Ethereum and Linea networks.
```

Comment:
why isn't this necessary anymore?

Reply:
@emilyJLin95 I think it may be due to some discussion you and Chris had about clarifying that Linea SDK == Postman SDK, and it seems the distinctions this paragraph draws could be unhelpful. 

Happy to adapt the paragraph rather than delete, though, if you still think we should clarify that they're the same thing.